.github/workflows/selfhosted.yml

```diff
@@ -59,4 +59,12 @@ jobs:
         pip install .[tiiq]
     - name: Test ${{ matrix.platform }}
       run: |
-        srun -p ${{ matrix.platform }} pytest -m qpu --no-cov --platforms ${{ matrix.platform }}
+        sed -i 's/$PLATFORM/'${{ matrix.platform }}'/' qputests.sh
+        srun -p ${{ matrix.platform }} qputests.sh
+        mkdir coverage
+        mv coverage.xml coverage/
+        mv htmlcov coverage/
```

Comment:
I think you also need to copy the htmlcov folder from /tmp/ to qibolab rootdir, in the qputests.sh script.

Reply:
You're definitely right, implemented in 1447f5b